terraform/e2e/main.tf

```diff
@@ -15,6 +15,7 @@
 locals {
   mapping_service_name = "mapping"
   policy_service_name  = "policy"
+  pmap_resource_scope  = "projects/pmap-dev"
```

Comment:
Overlooked this, we should make this an input variable. Once this change merged, we need to update https://github.com/abcxyz/infra/blob/main/terraform/legacy/pmap/dev/main.tf#L22

Reply:
Will do it after this. Created #139.

Reply:
This is an e2e terraform we recommend customer to use, better not hardcode anything us specific. It won't break the dev env since it has pined the past commit in infra. We can do it in the same PR, or leave this e2e module untouched, and fix it in the next PR.

Reply:
Updated, removed this local var, and made it a variable in variables.tf